pontoon/base/models.py

```diff
@@ -201,6 +272,10 @@ class Entity(models.Model):
     obsolete = models.BooleanField(default=False)
 
     @property
+    def unique_key(self):
+        return ':'.join([self.resource.path, self.string])
+
```

Comment:
Some file formats (xliff, properties, dtd, inc, ini, l20n) use "key" as a unique string identifier in a resource file.